tethysext/atcore/models/resource_workflow_steps/dataset_input_rws.py

```diff
@@ -0,0 +1,53 @@
+"""
+********************************************************************************
+* Name: form_input_rws.py
```

Comment:
Change "Name" to be:   dataset_input_rws.py

Reply:
Fixed this.

Reply:
This file looks like it was copied with the intent to create a new dataset_input_rws. In the end, though we called it table_input_rws.py and this file was not removed. I've removed the file.